amr-wind/wind_energy/actuator/turbine/fast/turbine_fast_ops.H

```diff
@@ -22,7 +22,7 @@ struct ReadInputsOp<TurbineFast, SrcTrait>
         auto& tdata = data.meta();
 
         // Get density value for normalization
-        pp.query("density", tdata.density);
+        pp.get("density", tdata.density);
```

Comment:
can you still add a check that this matches the incflo density as well? No reason not to have it and could avoid more issues.